apps/fxc-front/src/app/components/2d/path-element.ts

```diff
@@ -254,6 +250,63 @@ export class PathElement extends connect(store)(LitElement) {
     this.postScoreToHost(score);
   }
 
+  private computeScore(points: LatLon[]): Score {
+    const track: ScoringTrack = {
+      points: points.map((point, i) => {
+        return {
+          ...point,
+          alt: 0,
+          timeSec: i * 60,
+        };
+      }),
+      minTimeSec: new Date().getTime() / 1000,
+    };
+    const result = optimize({ track }, this.getLeague()).next().value;
```

Comment:
I need to review the rest but it sound like `optimize` should return a result rather a generator.

Reply:
In next PR, this code will be moved in `logic`part so that UI code will not have to bother with generators

Reply:
You can ask follow up in the issue I think now (I have assigned it to you)
Ping me if you still do not have permissions

Reply:
Ok I have the permissions. Issue updated: https://github.com/vicb/flyxc/issues/168#issuecomment-2136757341